dotnet/src/Experimental/Process.Abstractions/KernelProcessStepContext.cs

```diff
@@ -52,4 +55,22 @@ public ValueTask EmitEventAsync(
                 Visibility = visibility
             });
     }
+
+    /// <summary>
+    /// Emit an external event to through a <see cref="IExternalKernelProcessMessageChannel"/>
+    /// component if connected from within the SK process
+    /// </summary>
+    /// <param name="externalTopicName"></param>
+    /// <param name="processEventData"></param>
+    /// <returns></returns>
+    /// <exception cref="KernelException"></exception>
+    public async Task EmitExternalEventAsync(string externalTopicName, object? processEventData = null)
```

Comment:
Shawn suggested we remove the `external` naming. Maybe:

```suggestion
    public async Task EmitEventToTopicAsync(string topicName, object? processEventData = null)
```

Reply:
removed external